config/dev1/sys.config

```diff
@@ -36,8 +36,8 @@
   {aecore, [
       {sync_port, 3015},
 
-      {peers, [<<"aenode://pp_23YdvfRPQ1b1AMWmkKZUGk2cQLqygQp55FzDWZSEUicPjhxtp5@localhost:3025">>,
```

Comment:
these hardcoded addresses did not match the generated-on-startup peer keys, so nodes weren't able to connect

Reply:
i've noticed now that these are set at `aecore_suite_utils` so they shouldn't be changed, it's probably the `peer_key` and `peer_key.pub` that need to contain these keypairs